cyprecice/cyprecice.pyx

```diff
@@ -710,6 +710,42 @@ cdef class Participant:
 
         self.thisptr.setMeshTetrahedra (convert(mesh_name), cpp_vertices)
 
+    # remeshing
+
+
+    def reset_mesh (self, mesh_name):
+        """
+        Resets a mesh and allows setting it using set_mesh functions again.
+
+        Parameters
+        ----------
+        mesh_name : str
+            Name of the mesh to reset.
+
+        Notes
+        -----
+        This function is still experimental.
+        Please refer to the documentation on how to enable and use it.
+
+        Previous calls:
+            advance() been called
+
+        Examples
+        --------
+        Set mesh vertices for a 2D problem with 5 mesh vertices.
+
+        >>> positions = np.array([[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]])
+        >>> mesh_name = "MeshOne"
+        >>> vertex_ids = participant.set_mesh_vertices(mesh_name, positions)
+        >>> # later in the coupling loop
+        >>> if remeshing_required():
+        >>>     participant.reset_mesh(mesh_name)
+        >>>     positions = np.array([[1, 1], [3, 3], [5, 5]])
+        >>>     vertex_ids = participant.set_mesh_vertices(mesh_name, positions)
+        """
+
+        self.thisptr.resetMesh (convert(mesh_name))
+
```

Comment:
Shouldn't there be a `return` statement returning the new vertex IDs?

Reply:
Do you mean in the example?

The `resetMesh()` API only empties the mesh. There is no return value.